src/Util.vala

```diff
@@ -33,59 +33,14 @@ namespace Util {
         }
     }
 
-    public static bool map_source_action_transform_to_cb (Binding binding, Value from_value, ref Value to_value) {
-        Variant? variant = from_value.dup_variant ();
-        if (variant == null) {
-            warning ("Failed to Variant.dup_variant");
-            return false;
-        }
-
-        string map_source;
-        var val = variant.get_string ();
-        switch (val) {
-            case Define.MapSource.MAPNIK:
-                map_source = Shumate.MAP_SOURCE_OSM_MAPNIK;
-                break;
-            case Define.MapSource.TRANSPORT:
-                map_source = Shumate.MAP_SOURCE_OSM_TRANSPORT_MAP;
-                break;
-            default:
-                warning ("map_source_action_transform_to_cb: Invalid map_source: %s", val);
-                return false;
-        }
-
-        var registry = new Shumate.MapSourceRegistry.with_defaults ();
-        to_value.set_object (registry.get_by_id (map_source));
-
-        return true;
-    }
-
-    public static bool map_source_action_transform_from_cb (Binding binding, Value from_value, ref Value to_value) {
-        unowned var val = (Shumate.MapSource) from_value.get_object ();
-        string id = val.id;
-        switch (id) {
-            case Shumate.MAP_SOURCE_OSM_MAPNIK:
-                to_value.set_variant (new Variant.string (Define.MapSource.MAPNIK));
-                break;
-            case Shumate.MAP_SOURCE_OSM_TRANSPORT_MAP:
-                to_value.set_variant (new Variant.string (Define.MapSource.TRANSPORT));
-                break;
-            default:
-                warning ("map_source_action_transform_from_cb: Invalid map_source: %s", id);
-                return false;
-        }
-
-        return true;
-    }
-
-    public static static bool map_source_get_mapping_cb (Value value, Variant variant, void* user_data) {
```

Comment:
`static static`
I'm surprising that I made such fool mistake, this is not warned by valac, and nevertheless it works as normally :flushed: